metrics-datadog/src/main/java/org/coursera/metrics/datadog/transport/UdpTransport.java

```diff
@@ -22,6 +24,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(UdpTransport.class);
   private final StatsDClient statsd;
+  private final HashMap lastSeenCounters = new HashMap<String,Integer>();
```

Comment:
1. Metrics' Counter works with `long`, so we should store `Long`s in this map to avoid potential overflow.
2. Left-hand side is raw. Should be: `private final Map<String, Integer> lastSeenCounters = new HashMap<String, Integer>();`